TMessagesProj_App/build.gradle

```
@@ -3,6 +3,7 @@ apply plugin: 'com.android.application'
 repositories {
     mavenCentral()
     google()
+    maven { url 'https://www.jitpack.io' }
```

```suggestion
    maven {
            url "https://www.jitpack.io"
            content {
                includeModule 'com.github.UnifiedPush', 'android-connector'
            }
        }
```

To avoid other library to be fetched from jitpack